src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireServiceTest.java

```diff
@@ -26,262 +25,281 @@
+    Long id = 1L;
+    ProprietaireCreationDto proprietaireCreationDTO = ProprietaireCreationDto.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .build();
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(id);
+    when(proprietaireRepository.save(Mockito.any(Proprietaire.class))).thenReturn(proprietaire);
+    //ProprietaireDTO proprietaireDTO = proprietaireService.creerProprietaire(proprietaireCreationDTO);
+    ProprietaireUpdateDto proprietaireUpdateDTO = ProprietaireUpdateDto.builder()
+        .nom("Doe")
+        .build();
+    when(proprietaireRepository.findById(id)).thenReturn(Optional.of(proprietaire));
+    proprietaire.setNom(proprietaireUpdateDTO.getNom());
+    when(proprietaireRepository.save(any(Proprietaire.class))).thenReturn(proprietaire);
+
+    ProprietaireDto proprietaireDtoUpdated = proprietaireService.updateProprietaire(id,
+        proprietaireUpdateDTO);
+
+    Assertions.assertThat(proprietaireDtoUpdated).isNotNull();
+    Assertions.assertThat(proprietaireDtoUpdated.getId()).isEqualTo(id);
+    Assertions.assertThat(proprietaireDtoUpdated.getNom())
+        .isEqualTo(proprietaireUpdateDTO.getNom());
+    Assertions.assertThat(proprietaireDtoUpdated.getPrenom())
+        .isEqualTo(proprietaireCreationDTO.getPrenom());
+    Assertions.assertThat(proprietaireDtoUpdated.getEmail())
+        .isEqualTo(proprietaireCreationDTO.getEmail());
+    Assertions.assertThat(proprietaireDtoUpdated.getNumTel())
+        .isEqualTo(proprietaireCreationDTO.getNumTel());
+    Assertions.assertThat(proprietaireDtoUpdated.getDateInscription()).isEqualTo(localDateTime);
+    Assertions.assertThat(proprietaireDtoUpdated.getLogements()).isNull();
+
+  }
+
+  @Test
+  public void ProprietaireService_DeleteProprietaire_ReturnOneProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+    Long id = 1L;
+
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(id);
+    when(proprietaireRepository.findById(id)).thenReturn(Optional.of(proprietaire));
+    when(proprietaireRepository.save(any(Proprietaire.class))).thenReturn(proprietaire);
+    doNothing().when(proprietaireRepository).deleteById(id);
+
+    ProprietaireDto proprietaireDTO = proprietaireService.deleteProprietaire(id);
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireDTO' must contain no more than '1' consecutive capital letters.